skimage/segmentation/_felzenszwalb.py

```diff
@@ -52,6 +52,10 @@ def felzenszwalb(image, scale=1, sigma=0.8, min_size=20, multichannel=True):
     >>> segments = felzenszwalb(img, scale=3.0, sigma=0.95, min_size=5)
     """
 
+    if not multichannel:
```

Comment:
(which means you need to make sure you test that the ValueError is raised in the correct conditions, since all tests passed with this condition).